mesh/mesh.go

```diff
@@ -241,21 +257,34 @@ func (m *Mesh) ExtractUniqueOrderedTransactions(l *types.Layer) []*Transaction {
 	return MergeDoubles(txs)
 }
 
-func (m *Mesh) PushTransactions(oldBase types.LayerID, newBase types.LayerID) {
+func (m *Mesh) PushTransactions(oldBase, newBase types.LayerID) {
 	for i := oldBase; i < newBase; i++ {
-
 		l, err := m.GetLayer(i)
 		if err != nil || l == nil {
 			m.Error("could not get layer %v  !!!!!!!!!!!!!!!! %v ", i, err) //todo handle error
 			return
```

Comment:
I agree -- it's the same logic that I used [here](https://github.com/spacemeshos/go-spacemesh/pull/1316#discussion_r322730328) and the behaviour should be the same.

Since this is not related to this PR, let's create a ticket for this.